src/envo/envo-edit.owl

```diff
@@ -33524,9 +33545,14 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001570> ObjectSomeValuesFrom(<
 # Class: <http://purl.obolibrary.org/obo/ENVO_01001581> (sea surface layer)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001581> "A surface layer of a water body which is part of an ocean or sea.")
```

Comment:
>A surface layer of a water body which is part of an ocean or sea.

I'd say: "A surface layer which is part of an ocean or sea." 

This is a semantic ambiguity fix, as SSL is used to refer to the surfaces of oceans too. That should be added as a comment. 

In fact, even with seafloor and ocean floor, we should add related synonyms for the other term, with annotation properties on those synonyms (comments) saying they're there as these terms are used so loosely.